docs/grpc.md

```diff
@@ -1,428 +1,285 @@
-# gRPC Documentation
-
-
-
+# 🤖 gRPC API
```

Comment:
This file is auto generated and should not be modified by hand

Reply:
reverted changes